### DISCUSSION TITLE
Use kube-bench for cluster standardisation with custom checks

### DISCUSSION THREAD

**Discussion by tuwid** [aquasecurity/kube-bench#1107] [Questions and Help]
Hello,
currently, we have many clusters under management through argo but we'd want to have a checking mechanism to make sure we have standard components in all clusters ( ie prometheus, service monitors, logging etc ). 
Can I add custom checks in kube-bench to make this possible? What do you folks think would be the best way to do this? 

Thanks

**Comment by itaysk** [comment]
@tuwid could you please share more about what kind of custom checks do you need? are you familiar with our project Starboard? it is more flexible in terms of the tools/checks that it runs. 

If you're interested I would love to have a chat to understand what problems you are trying to solve and how one of our open source tools can help you.
